plugins/chat/assets/javascripts/discourse/lib/chat-live-panel.js

```diff
@@ -0,0 +1,26 @@
+import { tracked } from "@glimmer/tracking";
+import { action } from "@ember/object";
+import { TrackedArray } from "@ember-compat/tracked-built-ins";
+
+export default class ChatLivePanel {
```

Comment:
We could pass a reference to the actual DOM node for the panel in the constructor of this at some point, not sure -- most of the time we should be using ember to handle UI changes so not completely sold on the idea.

Reply:
I think we should avoid DOM by all means. Everything we can do through actual message model is less work to do and cleaner updates.